### ISSUE TITLE
GHC 9.2 support

### ISSUE THREAD

**Issue by amesgen** [emilypi/Base16#11]
```
src/Data/ByteString/Base16/Internal/Utils.hs:27:62: error:
    • Couldn't match expected type ‘Word#’ with actual type ‘Word8#’
    • In the first argument of ‘word2Int#’, namely ‘i’
      In the second argument of ‘indexWord8OffAddr#’, namely
        ‘(word2Int# i)’
      In the first argument of ‘W8#’, namely
        ‘(indexWord8OffAddr# alpha (word2Int# i))’
   |
27 | aix (W8# i) alpha = W8# (indexWord8OffAddr# alpha (word2Int# i))
   |                                                              ^
```

**Comment by emilypi** [comment]
That looks like a 9.2.1 error alright. I have a fix ready to go that I'll have a release out in a bit for, but since it's the latest GHC, i'm not rushing.

**Comment by kirelagin** [comment]
ummmm any news?

**Comment by emilypi** [comment]
@kirelagin Is that sass I detect?

**Comment by kirelagin** [comment]
No, sorry, I didn’t mean to come off as rude or anything like this. I was just genuinely confused, since you said you already had a fix, and I realise that 9.2.1 is the latest version, but, like, it has been released, so it seems pretty important for the ecosystem to support it (and most other packages already do).

**Comment by emilypi** [comment]
Fixed as of #11 and uploaded as https://hackage.haskell.org/package/base16-0.3.1.0 - thanks @kirelagin and @amesgen
